package.json

```diff
@@ -107,7 +107,7 @@
     "karma-detect-browsers": "^2.2.6",
     "karma-firefox-launcher": "^1.1.0",
     "karma-qunit": "^1.2.1",
-    "node-sass": "^4.7.2",
+    "node-sass": "^4.13.1",
```

Comment:
The outdated version of `node-sass` wasn't running on my local machine using Mojave.